modules/tinymce/src/themes/silver/main/ts/backstage/Backstage.ts

```diff
@@ -41,32 +41,66 @@ export interface UiFactoryBackstage {
   readonly setContextMenuState: (state: boolean) => void;
 }
 
-const init = (lazySink: () => Result<AlloyComponent, string>, editor: Editor, lazyAnchorbar: () => AlloyComponent): UiFactoryBackstage => {
+export interface UiFactoryBackstagePair {
+  readonly popup: UiFactoryBackstage;
+  readonly dialog: UiFactoryBackstage;
+}
+
+const init = (lazySinks: { popup: () => Result<AlloyComponent, string>; dialog: () => Result<AlloyComponent, string> }, editor: Editor, lazyAnchorbar: () => AlloyComponent): UiFactoryBackstagePair => {
   const contextMenuState = Cell(false);
   const toolbar = HeaderBackstage(editor);
-  const backstage: UiFactoryBackstage = {
+
+  const providers: UiFactoryBackstage['shared']['providers'] = {
```

Comment:
This should be using the `UiFactoryBackstageProviders` type instead if we're keeping things consistent.
```suggestion
  const providers: UiFactoryBackstageProviders = {
```

Reply:
Changed.